### PR TITLE
TC for [PolicyTableUpdate] Timeout to wait a response on PTU

### PR DESCRIPTION
**Pull request by istoimenova**
To run the script you need to replace
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag.lua
from #32. Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/013_ATF_P_TC_Timeout_to_wait_response_PTU.lua [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+function Test:TestStep_PTS_Timeout_wait_response_PTU()
+  local hmi_app_id = self.applications[config.application1.registerAppInterfaceParams.appName]
+  local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.appName } })
+  :Do(function(_,data)
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+
+      testCasesForPolicyTableSnapshot:create_PTS(true,
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/013_ATF_P_TC_Timeout_to_wait_response_PTU.lua]: @istoimenova create_PTS should be renamed as verify_PTS

File: test_scripts/Polices/Policy_Table_Update/013_ATF_P_TC_Timeout_to_wait_response_PTU.lua [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] Timeout to wait a response on PTU
+--
+-- Description:
+-- SDL should request PTU in case user requests PTU
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/013_ATF_P_TC_Timeout_to_wait_response_PTU.lua]: Please check description

File: test_scripts/Polices/Policy_Table_Update/013_ATF_P_TC_Timeout_to_wait_response_PTU.lua [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+-- 1. Used preconditions
+-- SDL is built with "-DEXTENDED_POLICY: EXTERNAL_PROPRIETARY" flag
+-- 2. Performed steps
+-- Application is registered.
+-- PTU is requested.
+--
+-- Expected result:
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/013_ATF_P_TC_Timeout_to_wait_response_PTU.lua]: Maybe it is good to have this as well:

> SDL->HMI:SDL.PolicyUpdate(file, timeout, retry[]) where :
> 'file' is a filepath to the stored PT snapshot*
> 'timeout' is the value from PTS "module_config" section, key <timeout_after_x_seconds>
> 'retry' is an array of values from "seconds_between_retries" section of Local PT

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**